test/data/rhel-upgrade/upgrade_verify.sh

```diff
@@ -67,6 +69,32 @@ tee /etc/osbuild-composer/repositories/rhel-9.5.json > /dev/null << EOF
     ]
 }
 EOF
+elif [[ ${VERSION_ID%.*} == 10 ]]; then
+  tee /etc/osbuild-composer/repositories/rhel-10.0.json > /dev/null << EOF
+{
+    "x86_64": [
+        {
+            "name": "baseos",
+            "baseurl": "http://download.devel.redhat.com/rhel-10/nightly/RHEL-10-Public-Beta/latest-RHEL-10.0/compose/BaseOS/x86_64/os/",
```

Comment:
Nitpick: there are already non-Beta RHEL-10.0 composes. Would it make sense to use those, instead of Beta? or do you want to explicitly test Beta?

Reply:
Hmmm, good question, thanks for the heads up. The ticket I'm working on RHELMISC-5648 specifies to test upgrade from 9.5 to 10.0-Beta but it might be obsolete at this time. I'll post a question on the ticket.

Reply:
Sure, this is fine from my PoV.